hadoop-hdds/hadoop-dependency-client/pom.xml

```diff
@@ -290,5 +294,10 @@ https://maven.apache.org/xsd/maven-4.0.0.xsd">
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>org.xerial.snappy</groupId>
+      <artifactId>snappy-java</artifactId>
+      <version>1.1.10.5</version>
```

Comment:
Please define a new property for the version, and add the dependency in `dependencyManagement`, both in root `pom.xml`.